### PR TITLE
Make gubernator show metadata from finished.json.

### PR DESCRIPTION
**Pull request by rmmh**

### CODE REVIEW

**Comment by spxtr** [comment]
@k8s-bot test this

**Comment by spxtr** [comment]
@k8s-bot test this

**Comment by spxtr** [comment]
@k8s-bot gubernator test this

File: gubernator/templates/build.html [hunk 1]
```diff
@@ -45,6 +45,11 @@
 			<tr><td>{{k}}<td>{{v}}
 			% endfor
 		% endif
+		% if finished and 'metadata' in finished
+			% for k, v in finished['metadata']|dictsort
+			<tr><td>{{k}}<td>{{v}}
```
Comment (ixdy) [inline gubernator/templates/build.html]: wait, we never close the tags?

html is fun.

Comment (rmmh) [inline gubernator/templates/build.html]: https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags

https://html.spec.whatwg.org/multipage/syntax.html#syntax-tag-omission

Comment (ixdy) [inline gubernator/templates/build.html]: #til

Comment (fejta) [inline gubernator/templates/build.html]: Yeah! This thread looks just like one Ryan had with me a few months ago :smirk_cat:

**Pull request merged**